jennchenn/uwexplore
# Delete from search cards

## Implementation description

- Base components to delete from search cards
- ~~Doesnt work quite yet~~
- After adding a course, will change to a delete button
- pressing delete brings up a modal to confirm the course deletion
- confirming will remove each instance of course from the calendar (all lec, tut, lab, etc)



## Steps to test

1. Search for one of the courses already on the calendar
2. Press the delete button
3. Confirm deletion
4. Verify that all sections corresponding to the course were deleted



## What should reviewers focus on?

- might hinder ability to add lec first, then add lab later 
- user would have to delete from cal, then go back and add both lec and lab together
- can change if we feel it's important
